src/components/Switch.jsx

```diff
@@ -0,0 +1,25 @@
+import React, { useState } from 'react';
+import '../switch.css';
+import Grade from './Grade';
+
+export const Switch = (props) => {
+  const { onLabel, offLabel, id } = props;
+
+  const [watched, setWatched] = useState(false);
+
+  const handleChange = (e) => {
+    setWatched(e.target.checked);
+  };
+
+  return (
+    <div style={{ display: 'flex', fontWeight: '400' }}>
+      <label style={{ fontSize: '12px' }}>{offLabel}</label>
+      <input className="react-switch-checkbox" id={`react-switch-${id}`} type="checkbox" onChange={handleChange} />
```

Comment:
Input is missing `checked` property. See docs on how to work with checkboxes: https://reactjs.org/docs/forms.html#handling-multiple-inputs